spec/lib/identity_config_spec.rb

```diff
@@ -29,4 +29,29 @@
       end
     end
   end
+
+  describe 'in_person_outage_message_enabled' do
+    it 'has valid config values for dates when outage enabled' do
+      if IdentityConfig.store.in_person_outage_message_enabled
+        expect(IdentityConfig.store.in_person_outage_expected_update_date).to_not be_empty
+        expect(IdentityConfig.store.in_person_outage_emailed_by_date).to_not be_empty
+
+        update_date = IdentityConfig.store.in_person_outage_expected_update_date.to_date
+        update_month, update_day, update_year =
+          IdentityConfig.store.in_person_outage_expected_update_date.remove(',').split(' ')
+        expect(update_month).to_not be_empty
+        expect(update_day).to_not be_empty
+        expect(update_year).to_not be_empty
+        expect(update_date).to_not be_nil
+
+        email_date = IdentityConfig.store.in_person_outage_emailed_by_date.to_date
+        email_month, email_day, email_year =
+          IdentityConfig.store.in_person_outage_emailed_by_date.remove(',').split(' ')
+        expect(email_month).to_not be_empty
+        expect(email_day).to_not be_empty
+        expect(email_year).to_not be_empty
+        expect(email_date).to_not be_nil
```

Comment:
this is more elegant but it will pass when we'd want it to fail. For example if we set a config value for one of the dates to `'October'` then no error will be raised, but we want to make sure all 3 parts of the date are present.